### ISSUE TITLE
Node initialization failed. Check connection settings.

### ISSUE THREAD

**Issue by mahalamobile** [dynexcoin/Dynex-Wallet-App#16]
on starting the gui wallet i get this, windows 10, any ideas?

**Comment by mahalamobile** [comment]
still this issue and no assistance, guys if u want people to use your project there needs to be an element of support, or at least acknowledgement of issues :)

**Comment by hitarusaiko** [comment]
I have the same problem.
In my case, I can use the wallet normally for a month (since early October).
Due to some issues, my HDD was full and Dynex wallet failed to sync... after that, I always find the same error warning as yours.
Tried reinstalling and I still receive the same error warning.

Temporary (but not ideal) solution:
Try to install Dynex Wallet in another computer, I can access the same wallet normally from another computer.

**Comment by mahalamobile** [comment]
yeah mine never worked from the get go, didnt even attempt to sync etc
